### PR TITLE
Walls are a slightly higher layer than floors

### LINKED ISSUE #12069: Walls built on resin floors will appear beneath them.
This can basically mean invisible walls which will confuse the hell out of anyone looking at it. I don't think this even used to happen, since they're both layer 2 and it should have the walls placed on top since it's newer, but what do I know?

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
So that floor is lava, and other weather which works based on an area overlay, does not cover walls when not intended

Fixes #12069

### CODE REVIEW

File: code/game/turfs/simulated/walls.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	density = 1
 	blocks_air = 1
 	explosion_block = 1
+	layer = 2.1
```
Comment (RemieRichards) [inline code/game/turfs/simulated/walls.dm]: ```TURF_LAYER + 0.1```

**Comment by WJohn** [comment]
Also should fix #12069

**Comment by ChangelingRain** [comment]
You'll need to change the cult turf overlays in overlays.dm so the cult wall overlay still shows up.
Are you really sure this won't break anything else? It seems kind of likely to do so.

**Comment by KorPhaeron** [comment]
>It seems kind of likely to do so.

Only one way to find out

**Comment by tkdrg** [comment]
> Only one way to find out

Testing it?

**Comment by KorPhaeron** [comment]
@tkdrg 

I meant recklessly merging and letting the server deal with it but yeah sure.

**Comment by tkdrg** [comment]
I checked the code.

This makes on the same layer as:
* AI holopads
* Most crayon decals
* planted glowshrooms

And it puts walls above:
* cult wallglow

My suggestion: Bump walls down to TURF_LAYER + 05, and make cult wallglow something more sensible than 2.01 (2.07 probably)

**Pull request merged**